cmd/timestamp-cli/app/verify.go

```diff
@@ -43,6 +47,10 @@ func addVerifyFlags(cmd *cobra.Command) {
 	cmd.MarkFlagRequired("timestamp") //nolint:errcheck
 	cmd.Flags().Var(NewFlagValue(fileFlag, ""), "cert-chain", "path to certificate chain PEM file")
 	cmd.MarkFlagRequired("cert-chain") //nolint:errcheck
+	cmd.Flags().String("nonce", "", "optional nonce passed with the request")
+	cmd.Flags().Var(NewFlagValue(oidFlag, ""), "oid", "optional oid passed with the request")
```

Comment:
Does this refer to the hash algorithm OID  ? If so, we might want to explain it in the description.

Reply:
If so, we could rename this flag to `hash-algorithm-oid`

Reply:
No, this refers to the TSA policy OID. I'll update the description to "optional TSA policy OID passed with the request".